src/test/java/api/support/fixtures/CalendarExamples.java

```diff
@@ -34,6 +34,8 @@
   public static final String CASE_PREV_OPEN_AND_CURRENT_NEXT_CLOSED = "85346678-2f09-4bc9-8924-3734882d44a3";
   public static final String CASE_CALENDAR_IS_EMPTY_SERVICE_POINT_ID = "66655555-2f09-4bc9-8924-3734882d44a3";
 
+  public static final String CASE_IN_ONE_DAY_IS_OPEN_NEXT_TWO_DAYS_CLOSED = "6ab38b7a-c889-4839-a337-86aad0297d7c";
```

Comment:
I think we can remove _IN from this name

Reply:
renamed